packages/supermassive/src/subscribeWithoutSchema.ts

```diff
@@ -0,0 +1,286 @@
+import { inspect } from "./jsutils/inspect";
+import { isAsyncIterable } from "./jsutils/isAsyncIterable";
+import { addPath, pathToArray } from "./jsutils/Path";
+import type { Maybe } from "./jsutils/Maybe";
+import {
+  GraphQLError,
+  locatedError,
+  Kind,
+  ASTNode as GraphQLASTNode,
+  TypeNode,
+} from "graphql";
+import { DocumentNode } from "./ast/TypedAST";
+
+import type { ExecutionContext } from "./executeWithoutSchema";
+import {
+  assertValidExecutionArguments,
+  buildExecutionContext,
+  buildResolveInfo,
+  executeWithoutSchema,
+  getOperationRootTypeName,
+} from "./executeWithoutSchema";
+import { collectFields } from "./collectFields";
+import { getArgumentValues } from "./values";
+import { typeNameFromAST } from "./utilities/typeNameFromAST";
+
+import { mapAsyncIterator } from "./utilities/mapAsyncIterator";
+
+import {
+  ObjectTypeResolver,
+  FunctionFieldResolver,
+  FieldResolver,
+  Resolvers,
+  ExecutionResult,
+} from "./types";
+
+export interface SubscriptionArgs {
```

Comment:
Are those different from execute args? Seems they are the same in graphql-js, let's unify it here too.

Reply:
We added `subscribeFieldResolver`

Reply:
You can add it to all ExecutionArgs, like in graphql-js.

https://github.com/graphql/graphql-js/blob/main/src/execution/execute.ts#L154